pkg/runtime/ref/injector.go

```diff
@@ -17,18 +17,40 @@
 package ref
 
 import (
+	"fmt"
+
 	"github.com/dapr/components-contrib/secretstores"
 
+	"mosn.io/layotto/components/configstores"
 	"mosn.io/layotto/components/ref"
 )
 
 type DefaultInjector struct {
-	Container RefContainer
+	//limit ref component
+	ComponentLimit map[string]struct{}
+	Container      RefContainer
+}
+
+var defaultInjector *DefaultInjector
```

Comment:
Looks like this variable is unused? Can we delete it?

![image](https://user-images.githubusercontent.com/26001097/192477492-ffe2a066-e0a4-4564-aa22-d00c3216eafd.png)